### PR TITLE
Made default viewport nicer on generated maps 

### PR DESCRIPTION
**Pull request by razvan-at-improbable**
This change doesn't affect maps on which the viewport isn't the default one. (e.g. maps with possession tests where the viewport follows a certain actor)

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp [hunk 1]
```diff
@@ -101,10 +104,12 @@ void UGeneratedTestMap::GenerateBaseMap()
...
 	// Default player start location is chosen so that players spawn on server 1 by default.
 	// Individual test maps can change this if necessary.
-	APlayerStart* PlayerStart = AddActorToLevel<APlayerStart>(CurrentLevel, FTransform(FVector(-100, -100, 100)));
-
-	// TODO: Maybe figure out how to set the default viewpoint when opening the map, so we don't start in the ground (maybe together with
-	// other viewing position issues), ticket: UNR-4975.
+	FTransform PlayerTransform;
+	FVector PlayerTranslation = FVector(-500, -500, 200);
+ 	PlayerTransform.SetTranslation(PlayerTranslation);
+	FRotator RotatorToOrigin = UKismetMathLibrary::MakeRotFromX(-PlayerTranslation);
```
Comment (improbable-danny) [inline SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp:110]: Seems a bit excessive to use Kismet here (I just pinched the implementation) and then the include can be removed 

```suggestion
	FRotator RotatorToOrigin = FRotationMatrix::MakeFromX(-PlayerTranslation).Rotator();
```

File: SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp [hunk 2]
```diff
@@ -37,6 +39,7 @@ AActor* UGeneratedTestMap::AddActorToLevel(ULevel* Level, UClass* Class, const F
 	return GEditor->AddActor(Level, Class, Transform);
 }
 
+
```
Comment (improbable-danny) [inline SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp:42]:
```suggestion
```

File: SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp (hunk 1 above)
Comment (mironec) [reply SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp:110]: Also maybe a small comment here, saying we want the player start to look at the origin (which is what this does).
If you want to be super-precise, you can also include a comment about the inability to take rotation around the Y axis (or whichever axis it was) into account.

File: SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp [hunk 3]
```diff
@@ -101,10 +104,12 @@ void UGeneratedTestMap::GenerateBaseMap()
...
-	APlayerStart* PlayerStart = AddActorToLevel<APlayerStart>(CurrentLevel, FTransform(FVector(-100, -100, 100)));
-
-	// TODO: Maybe figure out how to set the default viewpoint when opening the map, so we don't start in the ground (maybe together with
-	// other viewing position issues), ticket: UNR-4975.
+	FTransform PlayerTransform;
+	FVector PlayerTranslation = FVector(-500, -500, 200);
+ 	PlayerTransform.SetTranslation(PlayerTranslation);
+	FRotator RotatorToOrigin = UKismetMathLibrary::MakeRotFromX(-PlayerTranslation);
+	PlayerTransform.SetRotation(RotatorToOrigin.Quaternion());
+	APlayerStart* PlayerStart = AddActorToLevel<APlayerStart>(CurrentLevel, PlayerTransform);
```
Comment (improbable-danny) [inline SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/GeneratedTestMap.cpp:112]:
```suggestion
	AddActorToLevel<APlayerStart>(CurrentLevel, PlayerTransform);
```

**Review by improbable-danny — APPROVED** [review]
Two comments about a whitespace change and removing the Kismet header but otherwise looks good

**Pull request merged**